integration-tests/src/test/resources/bash-scripts/images.properties

```diff
@@ -101,3 +101,8 @@ middleware/fmw-infrastructure_cpu:12.2.1.3-jdk8-ol8;weblogick8s/test-images/fmw-
 #ghcr.io/verrazzano/fluentd-kubernetes-daemonset:v1.14.5-20230131231729-f85741b;weblogick8s/test-images/fluentd-kubernetes-daemonset:v1.14.5
 #k8s.gcr.io/ingress-nginx/controller:v1.2.0;weblogick8s/test-images/ingress-nginx/controller:v1.2.0
 #public.ecr.aws/docker/library/traefik:v2.9.6;weblogick8s/test-images/traefik-ingress/traefik:v2.9.6
+#quay.io/prometheus/alertmanager:v0.24.0;weblogick8s/test-images/prometheus/alertmanager:v0.24.0
+#quay.io/prometheus/node-exporter:v1.3.1;weblogick8s/test-images/prometheus/node-exporter:v1.3.1
+#quay.io/prometheus/pushgateway:v1.4.3;weblogick8s/test-images/prometheus/pushgateway:v1.4.3
+#grafana/grafana:9.3.0;weblogick8s/test-images/grafana/grafana:9.3.0
+#jimmidyson/configmap-reload:v0.5.0;weblogick8s/test-images/jimmidyson/configmap-reload:v0.5.0
```

Comment:
I can not pull jimmidyson/configmap-reload:v0.5.0 or grafana/grafana:9.3.0 directly using docker pull command.
Please provide the complete path for original location.

Reply:
you need to login first : 
[mkogan@mkogan-1 functional]$ docker pull grafana/grafana:9.3.0
Trying to pull repository docker.io/grafana/grafana ... 
9.3.0: Pulling from docker.io/grafana/grafana
Digest: sha256:659b34a5918d7ae6c2471e6141e03a985e508659bd58535263adf3170f7d15f8
Status: Image is up to date for grafana/grafana:9.3.0
grafana/grafana:9.3.0
[mkogan@mkogan-1 functional]$ docker pull jimmidyson/configmap-reload:v0.5.0
Trying to pull repository docker.io/jimmidyson/configmap-reload ... 
v0.5.0: Pulling from docker.io/jimmidyson/configmap-reload
Digest: sha256:904d08e9f701d3d8178cb61651dbe8edc5d08dd5895b56bdcac9e5805ea82b52
Status: Image is up to date for jimmidyson/configmap-reload:v0.5.0
jimmidyson/configmap-reload:v0.5.0

Reply:
On hosted linux box, we do not have permission to pull any images for docker.io. The whole purpose of the JIRA to avoid image pull from docker.io

Reply:
I found the following images in quay.io as follow ...
   docker  pull quay.io/image-mirror/docker.io/grafana/grafana
   docker pull quay.io/coreos/configmap-reload